firefox_ui_tests/functional/locationbar/test_star_in_autocomplete.py

```diff
@@ -47,6 +50,7 @@ def test_star_in_autocomplete(self):
 
         self.browser.menubar.select('Bookmarks', 'Bookmark This Page')
         done_button = self.marionette.find_element('id', 'editBookmarkPanelDoneButton')
```

Comment:
Please also add a TODO comment here that this need a new UI class for the editBookmarkPanel.